### PR TITLE
observability: structured logging for faucet requests

### PR DESCRIPTION
**Pull request by geekflyer**
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/4529)

### CODE REVIEW

File: crates/aptos-faucet/src/lib.rs [hunk 1]
```diff
@@ -195,21 +195,23 @@ pub fn routes(
...
+                forwarded_for = forwarded_for,
+                host = info.host(),
+                method = format!("{}", info.method()),
+                path = info.path(),
+                version = format!("{:?}", info.version()),
+                status = format!("{}", info.status()),
+                referer = info.referer(),
+                user_agent = info.user_agent(),
+                elapsed = info.elapsed(),
+                "mint request"
```
Comment (davidiw) [inline crates/aptos-faucet/src/lib.rs:214]: yuck, I really hate this type of logging, but I guess we only have this service anyway...

**Review by davidiw — APPROVED** [review]

File: crates/aptos-faucet/src/lib.rs (hunk 1 above)
Comment (geekflyer) [reply crates/aptos-faucet/src/lib.rs:214]: why though? can we at least agree that this is million times better than 10 random strings concatenated by spaces like it is right now? 😉

Comment (davidiw) [reply crates/aptos-faucet/src/lib.rs:214]: Haha, I did approve, didn't I?

Comment (davidiw) [reply crates/aptos-faucet/src/lib.rs:214]: It'd be good to get aligned with @banool on the logging since he will also have something for Tap.

Comment (banool) [reply crates/aptos-faucet/src/lib.rs:214]: I have an issue in tap to do structured logging too, so this works for me.

**Review by banool — APPROVED** [review]

**Pull request merged**